### ISSUE TITLE
Dropbox requires frequent remounting since latest chrome os update

### ISSUE THREAD

**Issue by tagerte** [yoichiro/chromeos-filesystem-dropbox#74]
Just in the past couple of days (I understand there may have been a Chrome OS update), my Dropbox is now disappearing from the Chrome files browser any time I'm inactive for more than a couple of hours. I have to log in to my dropbox account and mount the Dropbox again as if I've just installed the extension for the first time. Naturally this is very frustrating, and I love this extension so I'm hoping for a fix soon if possible!

**Comment by yoichiro** [comment]
@tagerte Hmm... I want to know more information.

* Is it unmounted after a few hours automatically?
* Is it unmounted after restarting your computer certainly?
* Is some error message shown at these time?

**Comment by tagerte** [comment]
As it turns out, I haven't encountered the issue again—on the day that I made this issue report (and the day before) the problem occurred twice or three times, often just when I had closed my laptop to put it to sleep. I did not restart my laptop on those days so I'm not sure if that causes the problem. No error message appears—the item "dropbox" just disappears from the Chromebook Files browser completely. 

I saw another poster on the google extension store reviews who was having the same issue on the same day. But I have noticed nothing since then— I'll stay alert and let you know if anything changes. Thanks!
